README.md

```diff
@@ -0,0 +1,42 @@
+# Schedule backup Utility
+This utility will fetch automatic/User created backups of the AWS resources by the user-defined tags, resources like RDS. And it can be used to schedule push backups to S3 bucket . 
```

Comment:
will it be a schedule type  utility ? 

I guess it will only push backups to s3 bucket